content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -162,9 +162,16 @@ Events: <none>
 
 ## Get the Certificate Signing Request Approved
 
-Approving the certificate signing request is either done by an automated
-approval process or on a one off basis by a cluster administrator. More
-information on what this involves is covered below.
+Approving the [certificate signing request](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/) is either done by an automated approval process or on a one off basis by a cluster administrator. Approving the request manually by using kubectl is illustrated below.
```

Comment:
nit:
```suggestion
Approving the [certificate signing request](/docs/reference/access-authn-authz/certificate-signing-requests/)
is either done by an automated approval process or on a one off basis by a cluster
administrator. If you're authorized to approve a certificate request, you can do that
manually using `kubectl`; for example:
```

Reply:
Change contexts as your suggestion. Thank you a lot!